swap/src/rendezvous.rs

```diff
@@ -0,0 +1,16 @@
+use std::fmt::{Display, Formatter};
+
+#[derive(Debug, PartialEq)]
+pub enum XmrBtcNamespace {
+    Mainnet,
+    Testnet,
+}
+
+impl Display for XmrBtcNamespace {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
```

Comment:
Inconsistent use of import vs fully-qualified. I always import `std::fmt` and qualify from there. Strikes a good balance between number of import lines, noise in the implementation and clarity of what is going on.